### PR TITLE
Adop 979

### PR DESCRIPTION
**Pull request by mohitvijay45**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/adoption/common/service/task/SendSubmissionNotifications.java [hunk 1]
```diff
@@ -14,11 +14,12 @@
...
 
+import static uk.gov.hmcts.reform.adoption.adoptioncase.model.State.AwaitingAdminChecks;
 import static uk.gov.hmcts.reform.adoption.adoptioncase.model.State.Submitted;
 
 @Component
 @Slf4j
-public class SendCitizenSubmissionNotifications implements CaseTask {
+public class SendSubmissionNotifications implements CaseTask {
 
     // @Autowired
```
Comment (pankajhmcts) [inline src/main/java/uk/gov/hmcts/reform/adoption/common/service/task/SendSubmissionNotifications.java:24]: Remove commented code @mohitvijay45

File: src/main/java/uk/gov/hmcts/reform/adoption/common/service/task/SendSubmissionNotifications.java [hunk 2]
```diff
@@ -45,6 +46,15 @@ public CaseDetails<CaseData, State> apply(final CaseDetails<CaseData, State> cas
...
+        if (EnumSet.of(AwaitingAdminChecks).contains(state)) {
+            log.info("Sending Local Authority application submitted notifications for case : {}", caseId);
+            try {
+                notificationDispatcher.sendToLocalAuthority(applicationSubmittedNotification, caseData, caseId);
+            } catch (NotificationClientException | IOException e) {
+                log.error("Couldn't send notifications");
+            }
+        }
+
         //log.info("Sending outstanding action notification if awaiting documents for case : {}", caseId);
```
Comment (pankajhmcts) [inline src/main/java/uk/gov/hmcts/reform/adoption/common/service/task/SendSubmissionNotifications.java:58]: Remove commented code or add log statements.

File: src/test/java/uk/gov/hmcts/reform/adoption/notification/ApplicationSubmittedNotificationTest.java [hunk 3]
```diff
@@ -29,14 +30,18 @@
 import static uk.gov.hmcts.reform.adoption.document.DocumentConstants.YES;
 import static uk.gov.hmcts.reform.adoption.notification.CommonContent.APPLICATION_REFERENCE;
 import static uk.gov.hmcts.reform.adoption.notification.CommonContent.SUBMISSION_RESPONSE_DATE;
+//import static uk.gov.hmcts.reform.adoption.notification.NotificationConstants.*;
```
Comment (pankajhmcts) [inline src/test/java/uk/gov/hmcts/reform/adoption/notification/ApplicationSubmittedNotificationTest.java:33]: Remove commented code

**Review by rajatkgupta29 — CHANGES_REQUESTED** [review]
Please revert items modified with respect to LOCAL_COURT. Check in overall application not only in YAML.

File: src/main/resources/application.yaml [hunk 4]
```diff
@@ -38,13 +38,15 @@ uk:
             SAVE_SIGN_OUT: ''
             APPLICATION_SUBMITTED: 'b60ab1ad-fe9c-4098-81a5-ad6c976b7de3'
             APPLICANT_APPLICATION_SUBMITTED: 'a0aa356f-8c6f-437c-b1b9-3c321048001d'
-            LOCAL_COURT_APPLICATION_SUBMITTED: '381cfc64-5016-4fe7-bd88-3ceeb163ce69'
```
[the remark this answers is not in the document]
Comment (mohitvijay45) [reply src/main/resources/application.yaml:41]: Added the property as suggested.

**Review by pankajhmcts — APPROVED** [review]

**Review by rajatkgupta29 — APPROVED** [review]

**Review by pankajhmcts — APPROVED** [review]

**Pull request merged**